tests/ceph_ansible/test_ansible_upgrade.py

```diff
@@ -146,6 +148,9 @@ def run(ceph_cluster, **kw):
     if config.get("ansi_cli_args"):
         cmd += config_dict_to_string(config["ansi_cli_args"])
 
+    if build.startswith("5.1"):
+        cmd += " -e qe_testing=true"
+
```

Comment:
Is this something new that is introduced? If yes, can we add documentation about it.

Reply:
This is a known issue while upgrading to RHCS 5.1. Since we had an upgrade blocker introduced there. If we need to workaround it, we need to add this extra argument. I will add doc text here.